api/users/views.py

```diff
@@ -103,7 +103,6 @@ class UserList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin):
     """
     permission_classes = (
         drf_permissions.IsAuthenticatedOrReadOnly,
-        base_permissions.RequiresScopedRequestOrReadOnly,
```

Comment:
So this class, `RequiresScopedRequestOrReadOnly`, needs to check if there's a cookie or appropriate token.  If so, allow to go through. If just basic auth, the request should fail?